nanofi/include/sitetosite/CPeer.h

```diff
@@ -88,7 +88,7 @@ static void setPort(struct SiteToSiteCPeer * peer, uint16_t port) {
   peer->_port = port;
   if(peer->_url != NULL) {
     int i;
-    for(i = strlen(peer->_url) -1; i >= 0; --i) {  // look for the last ':' in the string
+    for(i = (int)(strlen(peer->_url)) - 1; i >= 0; --i) {  // look for the last ':' in the string
```

Comment:
the google coding guideline suggests using `static_cast` here (some other places touched by this PR also use C-style casts in the old code)

Reply:
This is nanofi code, which is supposed to be in C.  This is not entirely true, as there is some C++ code in it already, but I did not want to add more.